semgrep/tests/e2e/test_performance.py

```diff
@@ -24,6 +24,8 @@ def test_debug_performance(run_semgrep_in_tmp):
 
```

Comment:
BTW @brendongo was it intended to run this python rule on the target file simple.yaml?
There will never be a match there.

Reply:
Yeah it doesn't actually need to run semgrep-core this test is verifying that the debug output doesnt cause a slowdown and since the rule has a really long message it would cause a slowdown if we handle stderr improperly. 

IMO we shouldn't allow the 30% slowdown